contracts/RelayHub.sol

```diff
@@ -294,7 +295,7 @@ contract RelayHub is IRelayHub {
             // acceptRelayedCall that this will not happen.
             (success, retData) = relayRequest.relayData.paymaster.call{gas:gasLimits.preRelayedCallGasLimit}(atomicData.data);
             if (!success) {
-                revertWithStatus(RelayCallStatus.PreRelayedFailed);
+                revertWithStatus(RelayCallStatus.PreRelayedFailed, bytes(""));
```

Comment:
any special reason you *don't* want to show the revert reason of preRelayedCall ?

Reply:
Not really. Fixing)